### PR TITLE
POL-37 POST provider oauth code

### PR DESCRIPTION
**Pull request by dvoet**

### CODE REVIEW

File: service/src/main/java/bio/terra/externalcreds/util/CompositeBackOffPolicy.java [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (snwessel) [inline service/src/main/java/bio/terra/externalcreds/util/CompositeBackOffPolicy.java]: Is this empty file supposed to be here?

File: service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java [hunk 2]
```diff
@@ -62,37 +62,47 @@ private String getUserIdFromSam() {
...
+    var userId = getUserIdFromSam();
+    var linkedAccount = linkedAccountService.getLinkedAccount(userId, provider);
     if (linkedAccount == null) {
       return ResponseEntity.notFound().build();
     }
-    val expTime =
+    var expTime =
         OffsetDateTime.ofInstant(linkedAccount.getExpires().toInstant(), ZoneId.of("UTC"));
-    val linkInfo =
+    var linkInfo =
```
Comment (snwessel) [inline service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java]: I don't think this variable is being used anymore (now that we're calling `getLinkInfoFromLinkedAccount` below).

**Review by snwessel — COMMENTED** [review]
This looks great overall, just noticed a couple of small things as I was looking through.

**Review by zarsky-broad — APPROVED** [review]
LGTMob

**Review by snwessel — APPROVED** [review]

**Pull request merged**